core/shared/src/main/scala/scalaz/zio/internal/StackBool.scala

```diff
@@ -27,30 +27,31 @@ private[zio] final class StackBool private () {
   private[this] var _size = 0L
 
   final def getOrElse(index: Int, b: Boolean): Boolean = {
-    var j   = index.toLong
+    val i0  = _size & 63L
+    val i   = ((64L - i0) + index) & 63L
```

Comment:
```suggestion
    val base = (64L - i0) + index
    val i    = base & 63L
    var ie   = base >> 6
```

Reply:
Thanks, I was struggling with the naming here :)